.github/workflows/os-increment-plugin-versions.yml

```diff
@@ -53,11 +53,8 @@ jobs:
           - {repo: opensearch-remote-metadata-sdk}
           - {repo: opensearch-learning-to-rank-base}
         branch:
-          - 2.x
-          - main
```

Comment:
We cans till keep `main` right Peter as it will handle future version increments?

Reply:
I will add main once 3.0.0 is out.